### PR TITLE
chore: Support platform selection on IntegrationKit

### PR DESCRIPTION
**Pull request by christophd**
- Do only wakeup matching integration kits when waiting for selected platform

### CODE REVIEW

File: pkg/controller/integrationkit/integrationkit_controller.go [hunk 1]
```diff
@@ -148,6 +148,11 @@ func add(_ context.Context, mgr manager.Manager, r reconcile.Reconciler) error {
 				}
 
 				for _, kit := range list.Items {
+					if v, ok := kit.Annotations[v1.PlatformSelectorAnnotation]; ok && v != p.Name {
+						// integration kit is waiting for another platform - skip it now
```
Comment (squakez) [inline pkg/controller/integrationkit/integrationkit_controller.go:152]: Would it make sense to log it by any chance?

**Review by squakez — APPROVED** [review]

**Review by oscerd — CHANGES_REQUESTED** [review]
I think logging would be useful

File: pkg/controller/integrationkit/integrationkit_controller.go (hunk 1 above)
Comment (oscerd) [reply pkg/controller/integrationkit/integrationkit_controller.go:152]: +1

Comment (christophd) [reply pkg/controller/integrationkit/integrationkit_controller.go]: Done

**Comment by christophd** [comment]
@oscerd @squakez I have added logging when skipping kits

**Comment by christophd** [comment]
@oscerd could you please (re)trigger the failed job? hopefully only a hiccup

**Comment by oscerd** [comment]
Done.

**Comment by squakez** [comment]
Please, rebase as we solved the github actions issue.

**Comment by christophd** [comment]
@squakez rebase done, could you please trigger the workflow once again 🙏

**Pull request merged**